src/components/TopBar.js

```diff
@@ -55,6 +55,10 @@ const styles = {
     }),
     icon: {
       marginRight: 12, flex: 'none'
+    },
+    navSection: {
+      flex: 'none', height: 70, padding: '0 28px', fontWeight: 600,
+      borderTop: `1px solid ${colors.dark(0.55)}`, color: 'white'
```

Comment:
So, I actually found an accessibility issue with `navSection` as well. If you go into the console, and type `configOverridesStore.set({isBioDataCatalyst: true})`, and then refresh the page (side note, you can use this to test out the different skins of the site), then you can see the background colors/rendering is different. I think for color here we need a ternary that detects which skin of the site we are using, similar to what you did for backgroundColor here: https://github.com/DataBiosphere/terra-ui/pull/2421/files#diff-c4ec3f6a5be9c7a24d3354c6aebf3544af26b150a4ce323fea36aa668cfd883cR323

Reply:
Yep, makes sense! Thanks for the testing tip, I wasn't sure how to test those other code paths.

Reply:
Ok, so I can get it into the BioCatalyst skin, but as far as I can tell the nav sections look fine to me. The background color of the menu is different, but it's still dark and the menu items in white text pop out with sufficient contrast.

I'm also seeing the same background image and the same not-quite-white background color still seems to match.

What are you seeing that needs to be fixed?

Reply:
I attached two screenshots here of the top bar. Note the text that I highlight.:
<img width="1422" alt="Screen Shot 2021-04-08 at 2 05 02 PM" src="https://user-images.githubusercontent.com/15351021/114075350-749c6680-9873-11eb-8931-0714b1115ad0.png">
<img width="1426" alt="Screen Shot 2021-04-08 at 2 05 10 PM" src="https://user-images.githubusercontent.com/15351021/114075351-749c6680-9873-11eb-97eb-3c411d4bbbc3.png">

Reply:
What are the other skins I can try out?

Reply:
Aha! Thanks

Reply:
Ok, it looks like this will have to be fixed on each page that uses it. In the Workspaces section it's already styling the breadcrumbs properly with a ternary. I only see the problem on the Billing page. Anywhere else I should be looking?

Reply:
If it is working in other pages (I'd try like 2 or 3), then we should be good to just fix this one. My guess is somewhere this one got missed. 

You can see all the skins that exist here: https://github.com/DataBiosphere/terra-ui/blob/dev/src/libs/colors.js#L21

However, the highest priority ones in my opinion are the default, and BioDataCatalyst. We ideally would get all of them, but BDC is the one that has the federal requirement.

Reply:
Ok, I just pushed an update to the Billing page, which brings the way it renders breadcrumbs in line with the way it's done elsewhere, such as on the Workspaces pages, and ensures that the font color contrasts with the background. It increases the scope of this PR a little bit, but it's only a few lines of code